foreshadow/transformers/core/wrapper.py

```diff
@@ -27,19 +25,37 @@ def _get_modules(classes, globals_, mname):  # TODO auto import all sklearn
         classes: A list of classes
         globals_: The globals in the callee's context
         mname: The module name
+        wrap: True to wrap the transformers.
 
     Returns:
         The list of wrapped transformers.
 
     """
+    # noqa: D202
+    def no_wrap(t):
```

Comment:
Added a comment above, it might make sense to move this utils?

Reply:
Maybe I'm mixing up which previous comment you're referring to, but I don't think we should. We can chat about this.